extension/src/test/suite/util.ts

```diff
@@ -172,6 +173,10 @@ export const buildDependencies = (
     'createFileSystemWatcher'
   ).returns(mockDisposable)
 
+  const mockCheckSignalFile = stub(FileSystem, 'checkSignalFile').resolves(
+    false
+  )
```

Comment:
[F] We don't gain much by reading the filesystem for each of these tests so I've mocked out the call.